.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: ruby
 sudo: false
 rvm:
-  - 2.2.6
-  - 2.3.3
-  - 2.4.0
+  - 2.2.8
+  - 2.3.5
+  - 2.4.2
```

Comment:
If we set these to only be the major and minor versions (e.g. `2.2`, `2.3` and `2.4`) will Travis CI pick the latest patch version of each?

Reply:
I remember trying this once and it didn't work, but let's try again! 😄

Reply:
Hmm, it kinda works but the Rubies it picks are one patchlevel behind the current ones (2.2.7 instead of 2.2.8, 2.3.4 instead of 2.3.5, and 2.4.1 instead of 2.4.2). Maybe if we could do a `rvm get stable` before... can't find out how though. 😞

Reply:
Travis CI is probably caching older versions. I think this is probably fine, eventually they'll catch up and that's better than us having to manually update them.